schema/daqconf/triggergen.jsonnet

```diff
@@ -11,6 +11,7 @@ local nc = moo.oschema.numeric_constraints;
 // A temporary schema construction context.
 local cs = {
   tc_type:         s.number(   "TCType",        "i4", nc(minimum=0, maximum=9), doc="Number representing TC type. Currently ranging from 0 to 9"),
+  tc_type_name:     s.string(   "TCTypeName"),
```

Comment:
this could use more information: type, default value, documentation. If we know the possible values, it could be changed to enum instead so it's easier to check and throw an error.